python/svix/api.py

```diff
@@ -549,8 +574,8 @@ async def get_stats(
             client=self._client,
             app_id=app_id,
             endpoint_id=endpoint_id,
-            since=options.since,
-            until=options.until,
+            since=ensure_tz(options.since),
+            until=ensure_tz(options.until),
```

Comment:
The None values would have been passed through here as-is previously so we can just "decorate" the options without guarding.